### PR TITLE
SAK-42262 Add proposed maxlength to tool title on Pageorder

### PR DESCRIPTION
**Pull request by bgarciaentornos**
https://jira.sakaiproject.org/browse/SAK-42262

### CODE REVIEW

File: site-manage/pageorder/tool/src/webapp/content/templates/PageList.html [hunk 1]
```diff
@@ -36,7 +36,7 @@ <h1 rsf:id="msg=page_title">page_title</h1>
 									<div class="item_edit_box" style="display: none;">
 										<div>
 											<label rsf:id="page-name-label"for="page-name-input">Title</label>
-											<input rsf:id="page-name-input" class="new_title" id="page-name-input"></input>
+											<input rsf:id="page-name-input" class="new_title" maxlength="20" id="page-name-input"></input>
```
Comment (adrianfish) [inline site-manage/pageorder/tool/src/webapp/content/templates/PageList.html]: Is this always 20?

Comment (bgarciaentornos) [reply site-manage/pageorder/tool/src/webapp/content/templates/PageList.html]: It was the proposed value by the Jira creator, and matches the max value on that other PR I made. It comes from this constant:

`private final static int MAX_TOOL_TITLE_LENGTH = 20;`

So I'd say yes...

Comment (adrianfish) [reply site-manage/pageorder/tool/src/webapp/content/templates/PageList.html]: In that case you should supply it, not hardcode it, like SiteAction does, imo.

Comment (bgarciaentornos) [reply site-manage/pageorder/tool/src/webapp/content/templates/PageList.html]: Ok, seems fair. It's still a different tool, so I can't use the same variable, which is not ideal.

**Pull request merged**